.github/workflows/ci.yml

```diff
@@ -146,3 +146,24 @@ jobs:
     - name: Create source and wheel dist
       run: |
         python -m build
+
+  server-package:
+    # This job checks if we can build our server package
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./server
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build the Docker image
+        run: |
+          docker buildx build -t basyx-python-sdk-http-server .
+      - name: Run container
+        run: |
+           docker run -d -p 8080:80 -v ./storage:/storage basyx-python-sdk-http-server
+      - name: Stop the container
+        run: |
+          docker ps -q --filter "ancestor=basyx-python-sdk-http-server" | xargs -r docker stop
```

Comment:
Also, this is a little more complicated than it needs to be. It'd be much better to name the container in the `run` step (`docker run --name basyx-python-server` and then simply call for `docker stop basyx-python-server`.